ee/ucp/kubernetes/storage/use-azure-disk.md

```diff
@@ -0,0 +1,237 @@
+on. 
+
+```
+/subscriptions/<subscriptionID>/resourceGroups/<resourcegroup>/providers/Microsoft.Compute/disks/<diskname>
+```
+
+You can now create Kubernetes Objects referring to this Azure Disk. The below
+example is using a Kubernetes Pod, however the same Azure Disk syntax can be
+used for Daemonsets, Deployments and Statefulsets. In the below example the
+Azure Disk Name and ID to reflect the manually created Azure Disk. 
+
+```
+$ cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mypod-azuredisk
+spec:
+  containers:
+  - image: nginx
+    name: mypod
+    volumeMounts:
+      - name: mystorage
+        mountPath: /data
+  volumes:
+      - name: mystorage
+        azureDisk:
+          kind: Managed
+          diskName: k8s_volume_1
+          diskURI: /subscriptions/<subscriptionID>/resourceGroups/<resourcegroup>/providers/Microsoft.Compute/disks/<diskname>
+EOF
+```
+
+## Dynamically Provisioning Azure Disks
+
+### Defining the Azure Disk Storage Class
+
+Kubernetes can dynamically provision Azure Disks using the Azure Kubernetes
+integration which was configured for you when UCP was installed. For Kubernetes
+to know which APIs to use when provisioning storage, Operators need to create
+Kubernetes Storage Classes specific to each storage backend. For more
+information on Kubernetes Storage Class, see [Storage
+Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/).
+
+In Azure there are 2 different types of Azure Disk that can be consumed by
+Kubernetes. Azure Disk Standard Volumes and Azure Disk Premium Volumes. For more
+information on their differences see [Azure
+Disks](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-types).
+
+Depending on your use case, you may choose to deploy 1 or both of the Azure Disk storage Classes (Standard and Advanced).
+
+To create a Standard Storage Class:
+
+```
+$ cat <<EOF | kubectl create -f -
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: standard
+provisioner: kubernetes.io/azure-disk
+parameters:
+  storageaccounttype: Standard_LRS
+  kind: Managed
+EOF
+```
+
+To Create a Premium Storage Class
+
+```
+$ cat <<EOF | kubectl create -f -
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: premium
+provisioner: kubernetes.io/azure-disk
+parameters:
+  storageaccounttype: Premium_LRS
+  kind: Managed
+EOF
+```
+
+To see which Storage Classes have been provisioned.
+
+```
+$ kubectl get storageclasses
+NAME       PROVISIONER                AGE
+premium    kubernetes.io/azure-disk   1m
+standard   kubernetes.io/azure-disk   1m
+```
+
+### Creating an Azure Disk with a Persistent Volume Claim
+
+After an Operator has created a Storage Class, they can then use Kubernetes
+Objects to dynamically provision Azure Disks. This is done using Kubernetes
+Persistent Volumes Claims
+[PVCs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#introduction).
```

Comment:
Should we wrap the `PVC` in `(` + `)` to set it off a bit from the sentence? Or a `For more
information on Kubernetes Persistent Volume Claims, see` type of deal?